tests/FunderMaps.Testing/Repositories/TestAnalysisRepository.cs

```diff
@@ -9,35 +9,45 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
-namespace FunderMaps.IntegrationTests.Repositories
+namespace FunderMaps.Testing.Repositories
 {
     /// <summary>
     ///     Mockup <see cref="AnalysisProduct"/> repository.
     /// </summary>
-    public class TestAnalysisRepository : TestObjectRepositoryBase<AnalysisProduct>, IAnalysisRepository
+    public class TestAnalysisRepository : IAnalysisRepository
     {
+        /// <summary>
+        ///     Datastore holding the entities.
+        /// </summary>
+        public DataStore<AnalysisProduct> DataStore { get; set; }
+
         /// <summary>
         ///     Create new instance.
         /// </summary>
-        public TestAnalysisRepository(ObjectDataStore dataStore) : base(dataStore) { }
+        public TestAnalysisRepository(DataStore<AnalysisProduct> dataStore)
+        {
+            DataStore = dataStore;
+        }
 
-        public Task<IEnumerable<AnalysisProduct>> GetAllInFenceAsync(Guid userId, INavigation navigation, CancellationToken token) => throw new NotImplementedException();
+        public Task<IEnumerable<AnalysisProduct>> GetAllInFenceAsync(Guid userId, INavigation navigation, CancellationToken token)
+            => throw new NotImplementedException();
 
         public async Task<AnalysisProduct> GetByExternalIdAsync(Guid userId, string externalId, ExternalDataSource externalSource, CancellationToken token)
```

Comment:
None of these should actually be async (you may assign this to me though)

Reply:
The 'easiest' solution is to make them await an empty task. The solution is sufficient. Returning an incomplete task is more reliable in most situations however performance is not a consideration when SUT.